blocks/hooks/default-autocompleters.js

```diff
@@ -11,15 +11,19 @@ import { addFilter } from '@wordpress/hooks';
 /**
  * Internal dependencies
  */
-import { userAutocompleter } from '../autocompleters';
+import { blockAutocompleter, userAutocompleter } from '../autocompleters';
 
 // Exported for unit test.
 export const defaultAutocompleters = [ userAutocompleter ];
 
-function setDefaultCompleters( completers ) {
+function setDefaultCompleters( completers, blockName ) {
 	if ( ! completers ) {
 		// Provide copies so filters may directly modify them.
 		completers = defaultAutocompleters.map( clone );
+		// Add blocks autocompleter for Paragraph block
+		if ( blockName === 'core/paragraph' ) {
```

Comment:
maybe `getDefaultBlockName` instead of hard coding the block name? Since `core-blocks` is meant to be separate.

Reply:
Good idea, updating 👍